pkg/standalone/run.go

```diff
@@ -85,16 +85,17 @@ func (meta *DaprMeta) newAppID() string {
 }
 
 func (config *RunConfig) validateResourcesPath() error {
-	dirPath := config.ResourcesPath
-	if dirPath == "" {
-		dirPath = config.ComponentsPath
+	dirPath := config.ResourcesPaths
+	if len(dirPath) == 0 {
+		dirPath = []string{config.ComponentsPath}
```

Comment:
Shall we add a info log here?

Reply:
@shubham1172 Do you mean deprecation message ? We are already logging a deprecation warning message in cmd/run.go- 
```
RunCmd.Flags().MarkDeprecated("components-path", "This flag is deprecated and will be removed in the future releases. Use \"resources-path\" flag instead")
```
I think we can avoid logging it again here. WDYT ?